Server/JokesOnYou.Web.Api/DTOs/TagReplyDto.cs

```diff
@@ -7,5 +7,11 @@ namespace JokesOnYou.Web.Api.DTOs
 {
     public class TagReplyDto
     {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public DateTime Created { get; set; }
+        public List<JokeReplyDto> Jokes { get; set; }
```

Comment:
Not sure if this is a good thing to do... Won't this result in all tags being retrieved AND all jokes attached to those tags (I'm not sure)? Wouldn't it be better to have a dedicated call for that in the repo?

Reply:
Yes that is what is happening at the moment. All tags with all jokes. I suppose that is a lot of unnecessary data. I actually saw the DTO and assumed that is what was planned

Reply:
> Yes that is what is happening at the moment. All tags with all jokes. I suppose that is a lot of unnecessary data. I actually saw the DTO and assumed that is what was planned

Lets say you wanna get all jokes in a category. Then you would pass down the category name because its basically a filter. So get tags i think would be just for managing tags or our client to know what tags are available so they can design the front end

Reply:
So do I need to make a new DTO? Or just not fetch the jokes from db.

What about the Owner UserReplyDTO?

Reply:
You can remove the Jokes property from this dto, and yeah dont get the jokes from the db since that will become a performance issue. 

If we wanna show the client who made this dto we could just have the username instead of the actual user